W3ChampionsStatisticService/Hero/HeroController.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+
+namespace W3ChampionsStatisticService.Heroes;
+
+[ApiController]
+[Route("api/hero")]
```

Comment:
Can you ensure that this response is a long cache duration?

Reply:
Done for 7 days, seems reasonable enough

Reply:
I can add versioning of some kind using [Vary](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Vary?

Reply:
Updated to include `Vary` for the response cache using the hashcode of the enum.